.github/workflows/tests.yml

```diff
@@ -53,6 +53,7 @@ jobs:
         PSQL_SKIPSQLCMD: true
         PSQL_TESTDBNAME: gct_dev_ci
         PSQL_SSLMODE: disable
+        CI_TEST: true
```

Comment:
CI is always set to true on GH and also looks to be the same on AppVeyor, so this can be removed:

https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables